SOE-2250: removed margin-bottom for hero banner at max 767.

cjwest opened the pull request:
# READY FOR REVIEW

# Summary
- This sets the margin-bottom to 0 at max 767 width

# Needed By (Date)
- Abracadabra sprint end

# Criticality
- Fixes broken stuff


# Steps to Test
- Switch to this branch
- May need to revert feature
- Navigate to /admission-aid
- Verify you see less margin bottom at width 767 and smaller
- Navigate to /admission-aid/financial-aid
- Verify it looks okay

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
Dev: https://stanfordits.atlassian.net/browse/SOE-2294
## Related PRs

## More Information
### Before screen shots
![bannerbefore01](https://user-images.githubusercontent.com/284440/31564940-dcfc2bfc-b019-11e7-982a-2b42720ebd25.png)
![bannerbefore02](https://user-images.githubusercontent.com/284440/31564952-e550e702-b019-11e7-92c4-c8f4f9bf6b59.png)

### After screenshots:
![bannerafter01](https://user-images.githubusercontent.com/284440/31564980-fc5f0b18-b019-11e7-8156-5de5e8469971.png)

![bannerafter02](https://user-images.githubusercontent.com/284440/31564927-cf604d02-b019-11e7-9180-95ec5f7a0f99.png)

## Folks to notify


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

cjwest committed:
SOE-2250: removed margin-bottom for hero banner at max 767.

cjwest committed:
Merge branch '7.x-2.x' of github.com:SU-SOE/stanford_soe_helper into 7.x-2.x-style-tweaks-hero-banner

cjwest committed:
SOE-2294: Adjustmets to fix image height

kbrownell commented:
@cjwest YAAAAY!!!!! Screenshots make this so easy to check.  I see exactly what you see when it comes to the margin bottom.  Do you have screenshots for what I should with the scss/components/_soe_images.scss logged-in breakpoint changes?

cjwest commented:
@kbrownell 

Before:

![bannerbefore03](https://user-images.githubusercontent.com/284440/31624065-4031f934-b256-11e7-93d9-dc6fa70601fe.png)

After

![bannerafter03](https://user-images.githubusercontent.com/284440/31624090-5676baea-b256-11e7-8aa5-7b33d432ff0c.png)

kbrownell commented on scss/components/_soe_images.scss:
```diff
@@ -54,9 +54,14 @@
-
       .logged-in & {
         height: calc(100vh - 235px);
+        @include breakpoint-max(small) {
```
@cjwest It sounds from both you and Andy like this change is not coming from a client request.  It seems small, but still takes quite a bit of time away from our work already estimated for this sprint, not only in creating and documenting the change, but also testing, releasing, and then re-testing that it does not affect anything else.  If you can revert this file and commit the resulting css, I'll go ahead and merge the changes related to the margin-bottom on hero banners.

cjwest commented:
Hi Kellie,


As I have been working with this client, they have trusted us and appreciated it when we have worked to make changes to make things better for the site. I think that is reflected in our relationship.


I'm sorry to learn that everything must come from a client request, and we can no longer fix things when we find it convenient to address.


Caryl

________________________________
From: kbrownell <notifications@github.com>
Sent: Monday, October 16, 2017 11:29:01 AM
To: SU-SOE/stanford_soe_helper
Cc: Caryl J Westerberg; Mention
Subject: Re: [SU-SOE/stanford_soe_helper] SOE-2250: removed margin-bottom for hero banner at max 767. (#128)


@kbrownell requested changes on this pull request.

________________________________

In scss/components/_soe_images.scss<https://github.com/SU-SOE/stanford_soe_helper/pull/128#discussion_r144929357>:

>        .logged-in & {
         height: calc(100vh - 235px);
+        @include breakpoint-max(small) {


@cjwest<https://github.com/cjwest> It sounds from both you and Andy like this change is not coming from a client request. It seems small, but still takes quite a bit of time away from our work already estimated for this sprint, not only in creating and documenting the change, but also testing, releasing, and then re-testing that it does not affect anything else. If you can revert this file and commit the resulting css, I'll go ahead and merge the changes related to the margin-bottom on hero banners.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/SU-SOE/stanford_soe_helper/pull/128#pullrequestreview-69654897>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AARXGNhL7YpXsuuM_qrJPUIFWHuqwRqvks5ss6BtgaJpZM4P2VVF>.

cjwest committed:
SOE-2294: removed fix for extra space below hero banner.

cjwest commented:
@kbrownell changes made. please review.

cjwest commented:
Hey all,


I'll just add to this, that our client expects that we'll be maintaining their site display in as polished a way possible. And with theming, we should always expect bugs / needed tweaks to arise (nature of the beast). So in fact, it is a client request.


That said, it totally makes sense that we may not have the amount of time we need to make all of the changes we'd like to include in our PR's, so if anything needs to get pushed, that's understandable. But the changes should be included and prioritized the following PR.


hope this is helpful info! thanks team!

-k

________________________________
From: Caryl J Westerberg <cjwest@stanford.edu>
Sent: Monday, October 16, 2017 3:51:40 PM
To: SU-SOE/stanford_soe_helper; SU-SOE/stanford_soe_helper
Cc: Mention; Kerri Augenstein
Subject: Re: [SU-SOE/stanford_soe_helper] SOE-2250: removed margin-bottom for hero banner at max 767. (#128)


Hi Kellie,


As I have been working with this client, they have trusted us and appreciated it when we have worked to make changes to make things better for the site. I think that is reflected in our relationship.


I'm sorry to learn that everything must come from a client request, and we can no longer fix things when we find it convenient to address.


Caryl

________________________________
From: kbrownell <notifications@github.com>
Sent: Monday, October 16, 2017 11:29:01 AM
To: SU-SOE/stanford_soe_helper
Cc: Caryl J Westerberg; Mention
Subject: Re: [SU-SOE/stanford_soe_helper] SOE-2250: removed margin-bottom for hero banner at max 767. (#128)


@kbrownell requested changes on this pull request.

________________________________

In scss/components/_soe_images.scss<https://github.com/SU-SOE/stanford_soe_helper/pull/128#discussion_r144929357>:

>        .logged-in & {
         height: calc(100vh - 235px);
+        @include breakpoint-max(small) {


@cjwest<https://github.com/cjwest> It sounds from both you and Andy like this change is not coming from a client request. It seems small, but still takes quite a bit of time away from our work already estimated for this sprint, not only in creating and documenting the change, but also testing, releasing, and then re-testing that it does not affect anything else. If you can revert this file and commit the resulting css, I'll go ahead and merge the changes related to the margin-bottom on hero banners.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/SU-SOE/stanford_soe_helper/pull/128#pullrequestreview-69654897>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AARXGNhL7YpXsuuM_qrJPUIFWHuqwRqvks5ss6BtgaJpZM4P2VVF>.

kbrownell commented:
Double checked that the compiled CSS still removes the margin-bottom for hero banners at max 767.  Thanks for filling me in on how to take a holistic view of changes; it sounds like Andy is creating or has created a ticket for this logged-in issue, so the work and intent are not lost.  I'll go ahead and merge this now!
